src/freenet/client/filter/GenericReadFilterCallback.java

```diff
@@ -177,7 +178,7 @@ public String processURI(String u, String overrideType, boolean forBaseHref, boo
 				// @see bug #2297
 				return path;
 			} else if (linkFilterExceptionProvider != null) {
-				if (linkFilterExceptionProvider.isLinkExcepted(uri)) {
+				if (linkFilterExceptionProvider.isLinkExcepted(uri) || path.equals(ImageCreatorToadlet.ROOT_URL)) {
```

Comment:
This is pretty much what the `LinkFilterExceptedToadlet` is about. Implement that in the `ImageCreatorToadlet` and have the method return `true` for paths that you want to be exposable.